### ISSUE TITLE
Support for Web Toon images

### ISSUE THREAD

**Issue by chrisk414** [Kareadita/Kavita#1310]
I'm new to this project and I would like to make this as my default Comic and Webtoon viewer.

Webtoon images are vertically long, 700x5000 px for example.
![1001_064](https://user-images.githubusercontent.com/752887/172421508-b448da17-02a5-4b23-aef2-36417b59783b.jpg)

To view this I have to manually change the scale to fit using Width every time I enter the viewer. Also the title pages needs to be displayed properly. Currently, it tries to display the whole image and it ends up squashed vertically.  
Is it possible to recognize Webtoon images and set the scale to fit the Width automatically?

Thank you very much for your support.

**Comment by majora2007** [comment]
We have a webtoon mode in the reader. All reader settings can be set in user preferences.

**Comment by chrisk414** [comment]
I see. I found the option. It's a global option and it changes the reading behavior for all. 
Is it possible to make it a Collection option?

Besides, the title screenshot is still compressed.  Hope webtoon gets some love.
Thanks.

**Comment by majora2007** [comment]
No that is not possible. It's global or an override. 

Cover images for wevtoons are what they are due to the size. You should override them via edit screen.

**Comment by chrisk414** [comment]
Exactly, I'm just saying to make the option overridable in the Collection menu.
